run.sh

```diff
@@ -135,6 +135,11 @@ function report() {
     report_polarion
 }
 
+function upgrade_submariner() {
```

Comment:
The name could mislead.
We are not upgrading the submariner.
We are updating the catalog source.

You can change the function name to something like: update_subm_catalog or update_submariner_catalog